environment.yml

```diff
@@ -24,5 +25,4 @@ dependencies:
   - pytest=7.2
   - pyfakefs=5.1
   - cachetools=5.3
-  - pip:
-    - openghg_defs @ git+https://github.com/openghg/supplementary_data
+  - openghg_defs=0.0.2
```

Comment:
Would it be possible for us to do something like:
```
   - pip:
      - openghg_defs @ git+https://github.com/openghg/openghg_defs
```
or would that not work for this?

Reply:
Yeah I think that's the better way to do it actually. I've just pushed those changes.